app/src/test/scala/org/alephium/explorer/api/model/BlockEntryTest.scala

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.api.model
+
+import java.math.BigInteger
+
+import scala.collection.immutable.ArraySeq
+
+import akka.util.ByteString
+
+import org.alephium.explorer.service.FlowEntity
+import org.alephium.protocol.Hash
+import org.alephium.protocol.model.{BlockHash, GroupIndex}
+import org.alephium.util.TimeStamp
+
+final case class BlockEntryTest(
```

Comment:
Question: `BlockEntryTest` seems the same as `BlockEntry`. Can we directly use `BlockEntry` for testing?

Reply:
`BlockEntryTest` has `transactions`, while `BlockEntry` hasn't to serve the api separately in another paginated endpoint